src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/food/DietCookie.java

```diff
@@ -31,6 +32,7 @@ public DietCookie(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipe
         super(itemGroup, item, recipeType, recipe);
     }
 
+    @Nonnull
     @Override
     public ItemConsumptionHandler getItemHandler() {
```

Comment:
formatting. Also not even relevant to the sound manager PR
```suggestion
    @Override
    public @Nonnull ItemConsumptionHandler getItemHandler() {
```